lib/cdo/delete_accounts_helper.rb

```diff
@@ -132,28 +128,37 @@ def remove_user_from_sections_as_student(user)
     Follower.with_deleted.where(student_user: user).each(&:really_destroy!)
   end
 
-  # Removes all information about the user pertaining to Pardot. This encompasses Pardot itself, the
-  # contact_rollups pegasus table (master and reporting), and the contact_rollups_daily pegasus table
-  # (reporting only).
-  # @param [Integer] The user ID to purge from Pardot.
-  def remove_from_pardot(user_id)
+  def remove_contacts(email)
+    @pegasus_db[:contacts].where(email: email).delete
```

Comment:
Should we strip and downcase the email address before using it in the `where` query method?